image-inspector.go

```diff
@@ -103,11 +103,42 @@ func generateRandomName() string {
 	return fmt.Sprintf("image-inspector-%016x", n)
 }
 
+func getAuthConfigs(dockercfg, username, token_file *string) *docker.AuthConfigurations {
```

Comment:
@enoodle not `token_file`, it's `password_file` (same for the rest of the code).